webapp/channels/src/components/suggestion/switch_channel_provider.tsx

```diff
@@ -781,14 +821,11 @@ export default class SwitchChannelProvider extends Provider {
             if (channel.id === currentChannel?.id) {
                 continue;
             }
-            let wrappedChannel = {channel, name: channel.name, deactivated: false};
+            let wrappedChannel: WrappedChannel = {channel, name: channel.name, deactivated: false};
             const member = myMembers[channel.id];
             if (member) {
                 wrappedChannel.last_viewed_at = this.getLastViewedAt(member, myPreferences, channel);
             }
-            if (member && channelType === Constants.MENTION_UNREAD) {
-                wrappedChannel.unreadMentions = collapsedThreads ? member.mention_count_root : member.mention_count;
```

Comment:
This field shouldn't be confused with `wrappedChannel.unread_mentions`. As far as I can tell, this spelling wasn't actually used anywhere else in this file, and it didn't seem like it was a typo